docs/usage/README.md

```diff
@@ -18,5 +20,5 @@ func gettingStarted() {
 }
 ```
 
-Every Grav instance gets a UUID so that it can can identify itself to other instances when meshing is in use. `grav.New` can take a set of [options](grav-instance-options.md), but for an in-process Grav instance, no options are needed.
+Every Grav instance gets a UUID so that it can can identify itself to other instances when meshing is in use. `grav.New` can take a set of [options](https://github.com/suborbital/grav/docs/usage/grav-instance-options.md), but for an in-process Grav instance, no options are needed.
```

Comment:
Broken link; an absolute link would also need the `blob/$BRANCH/` path components:

```suggestion
Every Grav instance gets a UUID so that it can can identify itself to other instances when meshing is in use. `grav.New` can take a set of [options](https://github.com/suborbital/grav/blob/main/docs/usage/grav-instance-options.md), but for an in-process Grav instance, no options are needed.
```

Though I believe you can use relative links in GitHub:

```suggestion
Every Grav instance gets a UUID so that it can can identify itself to other instances when meshing is in use. `grav.New` can take a set of [options](./grav-instance-options.md), but for an in-process Grav instance, no options are needed.
```

Reply:
Ooh, thanks! I didn't know about either of those.